experimental/dds/tree/src/test/utilities/IdCompressorTestUtilities.ts

```diff
@@ -464,12 +484,12 @@ export class IdCompressorTestNetwork {
 				rowCount += 1;
 			}
 
-			// A local count > 1 indicates that this ID was unified, as more than one client has a local ID for it
-			// in their session space.
+			// A local count === 0 indicates the ID was created as an eager final, and thus cannot have had an
+			// override to unify.
 			if (rowCount === this.sequencedIdLogs.size && localCount <= 1) {
```

Comment:
This whole block was both wrong and never ran, huh lol :(

Reply:
Correct, lol.